src/consts.rs

```diff
@@ -15,3 +17,4 @@ pub mod IEC {
 }
 
 pub const SECTOR_SIZE: usize = 512;
+pub const DATA_BLOCK_SIZE: Sectors = Sectors(2048);
```

Comment:
Please define ```DATA_BLOCK_SIZE``` in the only module where it is used. I've been trying to limit the contents of this file to globally used constants.